### PR TITLE
Fix build instructions in README.md

### PR DESCRIPTION
**Pull request by VibhuJawa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ Usage instructions go here.
...
 
-To contribute to this library, first checkout the code. Then create a new virtual environment:
-
-    cd crossfit
-    python -m venv venv
-    source venv/bin/activate
+To contribute to this library, first create a conda environment with the necessary dependencies:
+```
+cd crossfit
+mamba env create -f conda/environments/cuda_dev.yaml
```
Comment (sarahyurick) [inline README.md:24]: When trying to create the environment I'm getting the following error:
```
(base) syurick@ipp1-3303:~/scratch/crossfit$ mamba env create -f conda/environments/cuda_dev.yaml
conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
nvidia/linux-64                                             Using cache
nvidia/noarch                                               Using cache
pkgs/r/linux-64                                               No change
pkgs/main/linux-64                                            No change
pkgs/main/noarch                                              No change
pkgs/r/noarch                                                 No change
rapidsai/linux-64                                             No change
rapidsai/noarch                                               No change


Looking for: ['bandit', 'black', 'cuda-version=12.0', "cudf[version='>=24.4']", "cuml[version='>=24.4']", "cupy[version='>=12.0.0']", 'dev', "dask-cuda[version='>=24.4']", "dask-cudf[version='>=24.4']", 'flake8', 'isort', 'pip', "pylibraft[version='>=24.4']", 'pytest', 'pytest-benchmark', "pytest-cov[version='>=2']", "pytorch[version='>=2.0']", 'transformers', 'tokenizers', 'torchvision', "scikit-learn[version='>=0.23.1']", 'scipy', "setuptools[version='>=61.0.0']", 'sentence-transformers', 'pip==-=pytrec_eval']


Encountered problems while solving:
  - nothing provides requested dev
  - nothing provides requested pip ==- pytrec_eval
```

**Review by sarahyurick — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VibhuJawa) [reply README.md:32]: Should be resolved now: https://github.com/rapidsai/crossfit/pull/61/commits/18fe8b2322c497e9dc71a2e188726d39f8c01fe5, tested it locally

Comment (sarahyurick) [reply README.md:32]: Yes, works now!

**Review by sarahyurick — APPROVED** [review]
LGTM

**Pull request merged**